content/news/052/index.md

```diff
@@ -70,15 +67,14 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Announcements
 
+TODO: I'll be working on adding an email subscription this weekend, so I'll write about it here once that's done :) - Jan
+
 ## Game Updates
 
 ### [Untitled Pixel Wizards Game][pixel-wizards]
 
 {{ embed_video(type="video/mp4", src="untitled-pixel-wizards-game.mp4",
-caption="Enemies now perceive, pursue and attack.. and occasionally get burned to death.") }}
-
-[![A Hound chases a player, and its corpse ragdolls after it is burned to death by a fireball](untitled-pixel-wizards-game.gif)][pixel-wizards]  
-_Enemies now perceive, pursue and attack.. and occasionally get burned to death._
+caption="Enemies now perceive, pursue and attack.. and occasionally get burned to death.") }} 
```

Comment:
What’s this caption ? It seems great ; is it visible ?

Reply:
![image](https://github.com/rust-gamedev/rust-gamedev.github.io/assets/9047632/1d7a37b6-18d8-4a39-8810-22a207826682)
Yep, looks like this :)

Reply:
looking great ! I found more info here: https://github.com/rust-gamedev/rust-gamedev.github.io/pull/1529